### ISSUE TITLE
Layout secondary column, option to make content sticky

### ISSUE THREAD

**Issue by alexdover** [Shopify/polaris-react#2050]
## Feature request summary

I am using the layout component with the secondary column and it would great if there was an option to make the content of that column sticky so that it remained on the right hand side as I scroll down.

## Rationale

In my case the left hand column can get quite long and a summary is shown in the secondary right hand column.

**Comment by alexdover** [comment]
Hi @mbaumbach - thanks for posting that, I really appreciate it!

**Comment by mbaumbach** [comment]
@alexdover I didn't realize this until I saw #2149, but Polaris has a `Sticky` component. I refactored our `StickyLayoutSection` to look like this:

```jsx
import React from 'react';
import {
    Layout,
    Sticky
} from '@shopify/polaris'

export default ({children}) => (
    <Layout.Section secondary>
        <Sticky offset="2rem">{children}</Sticky>
    </Layout.Section>
);
```

This was functionality equivalent and meant I didn't need to reference the Polaris CSS classes. Thanks @tmlayton for the pointer there and let me know if there's a better way to use the `Sticky` component in this case.
